interface/forms/gad7/gad7.inc

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * gad-7.inc - common includes and constants for the gad-7 form
+ * version 1.0.0  July 2020
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    ruth moulton
+ *
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+ require_once("../../globals.php");
+ require_once("$srcdir/api.inc");
+ //use OpenEMR\Common\Csrf\CsrfUtils;    // security module
```

Comment:
if not using best to remove instead of comment

Reply:
thanks, i'll take it out